forge-gui/res/cardsfolder/upcoming/baral_and_kari_zev.txt

```diff
@@ -0,0 +1,14 @@
+Name:Baral and Kari Zev
+ManaCost:1 U R
+Types:Legendary Creature Human
+PT:2/4
+K:First strike
+K:Menace
+T:Mode$ SpellCast | ValidCard$ Instant,Sorcery | ValidActivatingPlayer$ You | ActivatorThisTurnCast$ EQ1 | NoResolvingCheck$ True | Execute$ TrigMayCast | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast your first instant or sorcery spell each turn, you may cast a spell with lesser mana value that shares a card type with it from your hand without paying its mana cost. If you don't, create First Mate Ragavan, a legendary 2/1 red Monkey Pirate creature token. It gains haste until end of turn.
+SVar:TrigMayCast:DB$ Play | Valid$ Card.nonLand+YouOwn | ValidSA$ Spell.cmcLTX | ValidZone$ Hand | WithoutManaCost$ True | Optional$ True | ImprintPlayed$ True | SubAbility$ DBTokenIfNotPlayed
```

Comment:
"shares a card type with it" part is missing

Reply:
Caught it at work this morning, fixed!